getting_started/kubernetes-overview.adoc

```diff
@@ -0,0 +1,105 @@
+:_content-type: ASSEMBLY
+[id="kubernetes-overview"]
+= Kubernetes overview
+include::_attributes/common-attributes.adoc[]
+:context: kubernetes-overview
+
+toc::[]
+
+Kubernetes is an open source orchestration tool developed by Google. You can run and manage container-based workloads by using Kubernetes. The most common Kubernetes use case is to deploy an array of interconnected microservices, building an application in a cloud native way. You can create Kubernetes clusters that can span hosts across on-premise, public, private, or hybrid clouds, but some configuration is required.
+
+Traditionally, applications were deployed on top of a single operating system. With virtualization, you can split the physical host into several virtual hosts. Working on virtual instances on shared resources is not optimal in terms of efficiency and scalability. Because a virtual machine (VM) consumes as many resources as a physical machine, providing resources to a VM such as CPU, RAM, and storage can be expensive. Also, you might see your application degrading in performance due to virtual instance usage on shared resources.
+
+.Evolution of container technologies for classical deployments
+image::247-OpenShift-Kubernetes-Overview.png[]
+
+To solve this problem, you can use containerization technologies that segregate applications in a containerized environment. Similar to a VM, a container has its own filesystem, vCPU, memory, process space, dependencies, and more. Containers are decoupled from the underlying infrastructure, and are portable across clouds and OS distributions. Containers are inherently much lighter than a fully-featured OS, and are light weight isolated processes that run on the operating system kernel. VMs are slower to boot, and are an abstraction of physical hardware. VMs run on a single machine with the help of Hypervisor.
+
+You can perform the following actions by using Kubernetes:
+
+* Sharing resources.
+* Orchestrating containers across multiple hosts.
+* Installing new hardware configurations.
+* Running health checks and self-healing applications.
+* Scaling containerized applications.
+
```

Comment:
@xenolinux I think that Kubernetes components can be its own conceptual module, which can help break up the overview a bit and allow for potential content resuse

Reply:
Done